source/logo/faviconLogoRunning.svg

```diff
@@ -0,0 +1,10 @@
+</defs>
+</svg>
```

Comment:
When I try to load this into sketch, I get

<img width="439" alt="Screen Shot 2019-04-26 at 11 06 24 AM" src="https://user-images.githubusercontent.com/3766511/56817353-6c2bfc00-6813-11e9-92a9-9926482e1747.png">

I think it's missing the image svg inside.

Reply:
Hmmm, I'm not sure what's the issue. They seem to be loading fine for me - tested on 2 computers.

![Screen Shot 2019-04-26 at 11 21 35 AM](https://user-images.githubusercontent.com/13473834/56818397-8cf55100-6815-11e9-960e-e93b9ede7c86.png)

Reply:
Apologies, I didn't realize that SVGs are not standardized in a way...

The tool I used originally was Figma, which seems to have SVG compatibility issues with other design tools. 

https://www.figma.com/blog/with-figmas-new-svg-exports-less-more/
https://spectrum.chat/figma/general/svg-export-issues~9053ed2f-9ec7-4116-80c3-1a9207511905

I'll recreate in sketch.